src/bindings/c/src/common.h

```diff
@@ -15,9 +15,10 @@
 // TODO: we need to catch ov::Exception instead of ie::Exception
 #include "details/ie_exception.hpp"
 
-#define CATCH_OV_EXCEPTION(StatusCode, ExceptionType) \
-    catch (const InferenceEngine::ExceptionType&) {   \
-        return ov_status_e::StatusCode;               \
+#define CATCH_OV_EXCEPTION(StatusCode, ExceptionType)        \
+    catch (const InferenceEngine::ExceptionType& e) {        \
+        std::cout << "Exception: " << e.what() << std::endl; \
```

Comment:
We shouldn't print anything from the library

Reply:
ok, let remove it.

Reply:
done